_includes/tutorials/produce-consume-lang/scala/code/src/main/resources/application.conf

```diff
@@ -0,0 +1,61 @@
+bootstrap.servers = "localhost:29092"
+bootstrap.servers = ${?BOOTSTRAP_SERVERS}
+
+schema.registry.url = "http://localhost:8081"
+schema.registry.url = ${?SCHEMA_REGISTRY_URL}
+
+producer {
+  client-config {
+    acks = all
+    cient.id = scala-tutorial
```

Comment:
```suggestion
    client.id = scala-tutorial
```

Reply:
pretty sure it's client.id :/ https://docs.confluent.io/current/installation/configuration/producer-configs.html#client.id

Reply:
Yep! please commit this one @DivLoic